app/controllers/references_controller.rb

```diff
@@ -35,7 +35,20 @@ def destroy
     end
   end
 
+  # GET /username/references/1/edit_abstract
+  def edit_abstract
```

Comment:
We'll talk about this, but look at the 7 standard RESTful routes: https://github.com/briankung/projectcredo/pull/152#discussion_r88381936

If you can accomplish what you need to through them, we should use one of the standard routes and actions. In this case, it would probably be `paper#edit`.